.env

```
@@ -0,0 +1 @@
+TEST_ENV=valor de test
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)